From f8bf6410954fcf82bf28852e0dba015c6b7f19dc Mon Sep 17 00:00:00 2001
From: Ondrej Kozina <okozina@redhat.com>
Date: Fri, 22 May 2015 14:48:28 +0200
Subject: [PATCH] lvmetad.c: ignore lvmetad global handle on disconnect

do not unset lvmetad global handle on disconnect. This is
hotfix for issue described in:
https://www.redhat.com/archives/linux-lvm/2015-May/msg00008.html

Reported-by: Christian Hesse <list@eworm.de>
---
 lib/cache/lvmetad.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/lib/cache/lvmetad.c b/lib/cache/lvmetad.c
index 68162b6..73a382c 100644
--- a/lib/cache/lvmetad.c
+++ b/lib/cache/lvmetad.c
@@ -39,7 +39,6 @@ void lvmetad_disconnect(void)
 	if (_lvmetad_connected)
 		daemon_close(_lvmetad);
 	_lvmetad_connected = 0;
-	_lvmetad_cmd = NULL;
 }
 
 void lvmetad_init(struct cmd_context *cmd)
-- 
2.4.2

